addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -346,15 +349,17 @@
                             <b>Start</b>
                         </div>
                         <div class="col-auto ps-0">
-                            <span t-out="event.date_begin_located"/>
+                            <span itemprop="startDate" t-out="event.date_begin_located"/>
```

Comment:
is the itemprop important here? open question, just maybe I am not seeing something